services/app-web/src/common-code/proto/healthPlanFormDataProto/unlockedHealthPlanFormDataSchema.ts

```diff
@@ -167,4 +167,5 @@ export const unlockedHealthPlanFormDataSchema = z.object({
     rateDateEnd: z.date().optional(),
     rateDateCertified: z.date().optional(),
     rateAmendmentInfo: rateAmendmentInfoSchema.optional(),
+    rateProgramIDs: z.array(z.string()).optional(),
```

Comment:
does this work if you drop the optional() here? I think we are going to decode this into an empty array every time so the array itself doesn't need to be optional.

Reply:
Looks like it does!